test-requirements.txt

```diff
@@ -2,6 +2,7 @@ coverage>=4.0
 flake8>=2.2.0
 mock>=1.0.1
 PyHamcrest>=1.8.5
+pip!=19.0.1,!=19.0,>=18.1
```

Comment:
pip issue should not fix in the feature commit.
Ryan already fix in another commit: https://github.com/emc-openstack/storops/pull/257/files